Agreements/Internal/S_D.txt

```diff
@@ -42,6 +42,17 @@ COPX:EGLF:*:DIKVU:*:*:London AC Worthing:London AC Dover:21000:*:DIKVU
 COPX:EGLK:*:DIKVU:*:*:London AC Worthing:London AC Dover:21000:*:DIKVU
 COPX:EGTF:*:DIKVU:*:*:London AC Worthing:London AC Dover:21000:*:DIKVU
 COPX:EGHL:*:DIKVU:*:*:London AC Worthing:London AC Dover:21000:*:DIKVU
+COPX:EGVO:*:DIKVU:*:*:London AC Worthing:London AC Dover:21000:*:DIKVU
+COPX:EGTD:*:DIKVU:*:*:London AC Worthing:London AC Dover:21000:*:DIKVU
+
+COPX:EGHH:*:KONAN:*:*:London AC Worthing:London AC Dover:19000:*:KONAN
+COPX:EGHI:*:KONAN:*:*:London AC Worthing:London AC Dover:19000:*:KONAN
+COPX:EGLF:*:KONAN:*:*:London AC Worthing:London AC Dover:19000:*:KONAN
+COPX:EGLK:*:KONAN:*:*:London AC Worthing:London AC Dover:19000:*:KONAN
+COPX:EGTF:*:KONAN:*:*:London AC Worthing:London AC Dover:19000:*:KONAN
+COPX:EGHL:*:KONAN:*:*:London AC Worthing:London AC Dover:19000:*:KONAN
+COPX:EGVO:*:KONAN:*:*:London AC Worthing:London AC Dover:19000:*:KONAN
+COPX:EGTD:*:KONAN:*:*:London AC Worthing:London AC Dover:19000:*:KONAN
```

Comment:
not all these a/c will route via KONAN

Reply:
Probably the better solution is to use LYD and then the transfer point is around LYD15 or so

Reply:
Thanks for that, changed it now.